exercises/gas/contracts/Gas.sol

```diff
@@ -73,7 +73,7 @@ contract GasContract {
         //         i++;
         //     }
         // }
-        require(msg.sender == administrators[4]);
+        // require(msg.sender == administrators[4]);
```

Comment:
I don't understand how the "Checks for admin" test still passes if we remove this require statement @zlumer